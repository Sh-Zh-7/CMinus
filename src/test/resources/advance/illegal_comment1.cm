int main() {
    /* missing back-slash *
    /* missing star and back-slash
}