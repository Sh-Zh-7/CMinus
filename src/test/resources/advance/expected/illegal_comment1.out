Error type A at Line 4: Illegal block comment! Did you miss the '*' or '*/'?
